CHANGELOG.rst

```diff
@@ -1,6 +1,22 @@
 `Unreleased <https://github.com/pace-neutrons/Euphonic/compare/v0.6.4...HEAD>`_
 ----------
 
+- Changes:
+
+  - The following deprecated features have been removed:
+
+    - The ``return_mode_widths`` argument in ``calculate_qpoint_phonon_modes/frequencies``
+      has been removed
+    - The ``eta_scale`` argument in ``calculate_qpoint_phonon_modes/frequencies``
+      has been removed
+    - The ``--weights`` command-line tool argument has been removed
+    - The ``x_label``, ``y_label``, ``y_min`` and ``y_max`` arguments to
+      ``plot_1d/2d`` have been removed
```

Comment:
For the matplotlib options and ``--weights`` it might be more reassuring at a glance to make it clear that these were aliases and the functionality is still there.

Reply:
Done!